### PR TITLE
feat: support multiple prop in gutenberg editor

### PR DESCRIPTION
**Pull request by dratwas**
In this PR I added the possibility to set multiple media when inserting to the block.
Related gutenberg-mobile PR wordpress-mobile/gutenberg-mobile#1331
CI is failing when creating PRs from fork so there is PR that is created based on this one - https://github.com/wordpress-mobile/WordPress-Android/pull/10556
To test:
At this moment we don't support gallery block so this change can not be tested without changes in gutenberg-mobile JS bundle. In the future, we will use this functionality to multiple selections of media for the gallery.

* set multiple prop to true in media-upload bloc
* add image/video block - multiple select should be enabled for selecting media from the device or WP Media
* the array of selected media should be passed to the callback function

### CODE REVIEW

**Comment by pinarol** [comment]
Is this PR ready for review? If so, could you please add some description? @dratwas

**Comment by pinarol** [comment]
@mkevins could you help @dratwas add proper labels to this PR?

**Comment by dratwas** [comment]
Hi all :) This PR is ready to review.
CI is failing on `Copy Secrets` job but TBH I have no idea how I can fix that.

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java [hunk 1]
```diff
@@ -1600,14 +1603,12 @@ private void onUploadProgress(MediaModel media, float progress) {
         }
     }
 
-    private void launchPictureLibrary() {
-        // don't allow multiple selection for Gutenberg, as we're on a single image block for now
-        WPMediaUtils.launchPictureLibrary(this, !mShowGutenbergEditor);
+    private void launchPictureLibrary(boolean allowMultipleSelection) {
```
[the remark this answers is not in the document]
Comment (dratwas) [reply WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java]: Yes, you're right I can use `mAllowMultipleSelection` here 👍

**Pull request merged**